core_contracts/rewards/RewardData.py

```diff
@@ -151,3 +151,14 @@ def new_source(self, _name: str, _address: Address) -> None:
         source.name.set(_name)
         source.day.set(self._rewards._get_day())
         source.contract_address.set(_address)
+
+    def remove_source(self, _name: str, _address: Address) -> None:
+        source = self.__getitem__(_name)
```

Comment:
Check if _name exists in self._names

Reply:
used not in to check. 
let me know if this will create trouble.